### DISCUSSION TITLE
Console command doesnt read .env.local

### DISCUSSION THREAD

**Discussion by nsetyo** [symfony/symfony#49908] [Q&A] [answered]
I change my .env.local APP_ENV to prod but when i run comman `bin/console about` environment still in dev

Is this intentional?

**Comment by ghost** [comment, marked as answer]
too hard to answer with no details. Did you modify bin/console file? Do you have .env.dev.local file with APP_ENV=dev? Do yo have APP_ENV already set in your environment (e.g. in docker environment). 
Give more details. See what files are parsed in `Symfony\Component\Dotenv\Dotenv` in method that bin/console file calls.

**Reply by nsetyo** [reply]
Thanks for pointing me to the right direction. After debugging I know it is because of env in my container
